fmriprep/workflows/anatomical.py

```diff
@@ -74,43 +74,44 @@ def t1w_preprocessing(settings, name='t1w_preprocessing'):
             testing=settings.get('debug', False),
             template='mni_icbm152_nlin_asym_09c'
         ),
-        name='T1_2_MNI_Registration'
+        name='t1_2_mni'
     )
     # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
     # 6. FreeSurfer reconstruction
     if settings['freesurfer']:
-        surface_recon = surface_reconstruction(settings=settings)
+        surface_recon = surface_reconstruction(name='surface_recon',
```

Comment:
1. Sounds good
2. Seems reasonable
3. Then wouldn't the variable have the same name as the generator? `x_wf = x_wf(name="x_wf")`
4. Do you have an example of a sub-class workflows? I think I've only ever seen function-generated, so it'd need to be a pretty compelling improvement to break with standard usage.
5. It is a number of rules. I'd think of them more as heuristics, nothing hard and fast. It just tries to keep an orthographic link between the workflow name and the variable name, while not ending up with ridiculously long names or running into namespace collisions (see 3).